README.md

```diff
@@ -214,91 +218,92 @@ The API is used for implement safety offline transaction:
 ```js
 const tx = new LemoTx({from: 'Lemo836BQKCBZ8Z7B7N4G4N4SNGBT24ZZSJQD24D', to: 'Lemo83BYKZJ4RN4TKC9C78RFW7YHW6S87TPRSH34', amount: 100})
 const signedTxStr = tx.signWith('0xfdbd9978910ce9e1ed276a75132aacb0a12e6c517d9bd0311a736c57a228ee52')
-console.log(signedTxStr)
+console.log(signedTxStr.toString())
 // {"type":"0","version":"1","chainID":"1","gasPrice":"3000000000","gasLimit":"2000000","amount":"100","expirationTime":"1560244840","from":"Lemo836BQKCBZ8Z7B7N4G4N4SNGBT24ZZSJQD24D","to":"Lemo83BYKZJ4RN4TKC9C78RFW7YHW6S87TPRSH34","sigs":["0x55fe70309bb74aaad62a7fe4ab4085dd8c8bd450ce9eab8dd7906cc5453cbaab500f50e1d05ff746248bc806f4738be2fcaafc78a557edf1e34c976a21d6f0c200"],"gasPayerSigs":[]}
 ```
 
 ---
 
 ### static API
 
-<a name="LemoTx-signVote"></a>
-#### LemoTx.signVote
+<a name="tx-createVote"></a>
+#### LemoTx.createVote
 ```
-tx.signVote(privateKey, txInfo)
+LemoTx.createVote(txInfo)
 ```
-Sign a transaction for vote and return the signed transaction string  
-The API fills some special information in data, and creates a LemoTx [`instance`](#tx-constructor), then sign it.
+1. Create a transaction for vote  
```

Comment:
Create an unsigned LemoTx [`instance`](#tx-constructor) for vote
The API fills some special information in the `data` filed in LemoTx instance